src/plugins/visualizations/kibana.json

```diff
@@ -3,7 +3,5 @@
   "version": "kibana",
   "server": true,
   "ui": true,
-  "requiredPlugins": [
-    "expressions"
-  ]
+  "requiredPlugins": ["data", "expressions", "uiActions", "embeddable", "usageCollection"]
```

Comment:
Did not check each usages, but I have the feeling some of these could become optional dependencies instead? `usageCollection` mostly.